.env.example

```
@@ -0,0 +1,56 @@
+OPENAI_TEMPERATURE=0.7
+
+# Spring AI - Google Gemini Configuration
+GOOGLE_API_KEY=your-google-api-key-here
+GOOGLE_CLOUD_PROJECT_ID=your-google-cloud-project-id
+GOOGLE_CLOUD_LOCATION=asia-northeast3
+GEMINI_MODEL=gemini-2.0-flash-exp
+GEMINI_TEMPERATURE=0.7
+
+# Weather API Configuration
+WEATHER_API_KEY=your-weather-api-key-here
+WEATHER_API_BASE_URL=https://api.openweathermap.org/data/2.5
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The WEATHER_API_BASE_URL key should go before the WEATHER_API_KEY key

(UnorderedKey)